webapp/src/js/bootstrapper/index.js

```diff
@@ -138,17 +159,22 @@
           })
           .then(function() {
             // replication complete - bootstrap angular
-            $('.bootstrap-layer .status').text('Starting app…');
+            setUxStatus(BootstrapTranslator.LOAD_RULES);
```

Comment:
I am using it twice on purpose. One scenario is *Replication is not required, begin next phase*. The other is *Replication is required, I'm done, begin next phase*.

From my comments above above `Loading Rules...`

> Appears after successful sync or when sync is not required. We may consider changing this to "Initialising..." since it now captures the whole application initialisation instead of just the rules.

So as I see it -- we can either localize this in Angular and keep it as `Loading Rules...`, or move it here and rename it to "Initialising..."